jdaviz/configs/specviz2d/specviz2d.yaml

```diff
@@ -5,12 +5,14 @@ settings:
     toolbar: true
     tray: true
     tab_headers: false
+  dense_toolbar: false
```

Comment:
this might have implications for the MAST embed.

Reply:
@havok2063 , is this setting change a problem for you?